tests/test_interface_parser.py

```diff
@@ -181,29 +181,34 @@ def test_argument_list_templated(self):
     def test_default_arguments(self):
         """Tests any expression that is a valid default argument"""
         args = ArgumentList.rule.parseString(
-            "string c = \"\", string s=\"hello\", int a=3, "
-            "int b, double pi = 3.1415, "
-            "gtsam::KeyFormatter kf = gtsam::DefaultKeyFormatter, "
-            "std::vector<size_t> p = std::vector<size_t>(), "
-            "std::vector<size_t> l = (1, 2, 'name', \"random\", 3.1415)"
+            """string c = "", int z = 0, double z2 = 0.0, bool f = false, """
```

Comment:
Simply use backslashes rather than add all those extra quotes? There's no rule that says we can't use escape characters when needed.

Reply:
You're right.  Replaced with a single multiline string, how's that?

I just find escaped characters difficult to read, which probably I shouldn't have changed code based on personal preference, but now with multiline comment I think that's actually maybe the most appropriate and readable?